clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2276,6 +2337,141 @@ private Node leader(PartitionInfo partitionInfo) {
         return partitionInfo.leader();
     }
 
+    @Override
+    public void describeTopics(
+        TopicCollection topics,
+        DescribeTopicsOptions options,
+        AdminResultsSubscriber<DescribeTopicPartitionsResult> subscriber) {
```

Comment:
```suggestion
    public void describeTopics(
        TopicNameCollection topics,
        DescribeTopicsOptions options,
        AdminResultsSubscriber<DescribeTopicPartitionsResult> subscriber) {
```

This makes more sense in that `TopicNameCollection` is really all we support for this release. We could later generalize the API method signature to accept `TopicIdCollection` once support is added, and it shouldn't break any existing usage.